From a7cc42d115fcad07d264e00e7911f8c087dfc632 Mon Sep 17 00:00:00 2001
From: YDheeraj <yadavdheeraj2002@gmail.com>
Date: Fri, 25 Aug 2023 11:34:57 +0000
Subject: [PATCH 2/2] changes in branch b1 in main

---
 main.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/main.cpp b/main.cpp
index ca02a6f..e54c075 100644
--- a/main.cpp
+++ b/main.cpp
@@ -8,6 +8,9 @@ for(int i=0;i<5;i++){
 
 }
 
+for(int i=0;i<4;i++){
+ cout<<"for the change in new branch"<<endl;
+}
 
 
 return 0;
-- 
2.37.3

